packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -221,6 +221,54 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
   }
 }
 
+/// Tool bar configuration for [EditableText].
```

Comment:
Maybe write this "Toolbar" instead of "Tool bar" (here and below)?  I think putting a space in between implies that the B should be capitalized when camel cased ("ToolBar"), but it's not.

This is a definite nitpick though, totally up to you :)